pilot-app/src/components/page/explorer.vue

```diff
@@ -0,0 +1,25 @@
+﻿<template>
+    <section class="component explorer" v-if="content">
+        <div class="widget" v-for="folder of content" :key="folder.id">
+            <h2>
+                <a :href="folder.folder.link.href" v-html="folder.folder.link.text"></a>
+            </h2>
+            <ul>
+                <li v-for="listItem in folder.listItems">
+                    <a :href="listItem.link.href" v-html="listItem.link.text" :title="listItem.link.text"></a>                    
+                </li>
+            </ul>
+        </div>
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 8 spaces.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8170341)
```suggestion
    </section>
```